### PR TITLE
CP-15402: New Home Page (html based)

### PR DESCRIPTION
**Pull request by GaborApatiNagy**
Replaced existing to a new, html-based Home Page

### CODE REVIEW

**Comment by GaborApatiNagy** [comment]
Note, that this is not the final version, it doesn't need to be merged yet.

**Comment by stephen-turner** [comment]
Gabor, I haven't looked at the code in detail, but I'm wondering why you deleted HomePage.cs and added HomePageWeb.cs. I would prefer to reuse HomePage.cs.

Also, can we call the directory with the files in HomePage rather than Home, please. Thank you.

**Comment by MihaelaStoica** [comment]
Gabor, can we please put all the test from the License file in the Legal Notices dialog, and than we won't need to include this file.

**Comment by GaborApatiNagy** [comment]
@stephen-turner @MihaelaStoica Thanks for the comments, I've done all the changes needed. I dropped some commits in order to keep the history of the HomePage.cs (and related) files.

File: XenAdmin/HomePage/License [hunk 1]
```diff
@@ -0,0 +1,23 @@
+    Blueprint CSS Framework License
```
Comment (stephen-turner) [inline XenAdmin/HomePage/License]: I don't think we need this file if we have it in Legal Notices.

File: XenAdmin/Properties/Resources.resx [hunk 2]
```diff
@@ -127,9 +127,6 @@
   <data name="sl_add_storage_system_16" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Images\sl_add_storage_system_16.png;System.Drawing.Bitmap, System.Drawing, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
-  <data name="homepage_background" type="System.Resources.ResXFileRef, System.Windows.Forms">
```
Comment (stephen-turner) [inline XenAdmin/Properties/Resources.resx]: If you're going to remove these images from the resources, you can remove them from the source tree too.

File: WixInstaller/XenCenter.wxs [hunk 3]
```diff
@@ -80,6 +80,8 @@
                             <File Id="DiscUtils" Source="..\XenAdmin\bin\Release\DiscUtils.dll" />
                             <File Id="XenOvf" Source="..\XenAdmin\bin\Release\XenOvf.dll" />
                             <File Id="XenOvfTransport" Source="..\XenAdmin\bin\Release\XenOvfTransport.dll" />
+							<!-- Homepage -->
+	                        <File Id="" Source="..\XenAdmin\bin\Release\HomePage\index.mht" />
```
Comment (stephen-turner) [inline WixInstaller/XenCenter.wxs]: To think about: is this the right location to pick this file up from?

**Comment by stephen-turner** [comment]
Apart from these small comments, the changeset looks good: very neat.

**Comment by GaborApatiNagy** [comment]
Updated PR, please see my latest commit for the changes. Thank you

**Comment by stephen-turner** [comment]
@GaborApatiNagy, I'm pulling these changes, but you still haven't addressed the request to remove no-longer-used images from the source tree. Please do that in a separate PR. Thanks.

**Pull request merged**